tests/robustness/linearizability_test.go

```diff
@@ -70,6 +72,22 @@ var (
 			},
 		},
 	}
+	ReqProgTraffic = trafficConfig{
```

Comment:
No need for a separate traffic, I think we can enable requestProgress on default traffic.

Reply:
Not sure... Requesting progress notifications on every event is rather untypical behaviour, so I felt having it enabled more broadly might end up shadowing other bugs, or at least make debugging them messier. So it felt worth it to put in the extra effort to construct a specific traffic use case.

If you insist I can revert to a simpler approach. Would you suggest I also remove the traffic flag then, and just hard-code it into `watchMember`? That seemed like non-obvious behaviour to me...

Reply:
That's a valid point, let's leave it as it is and revisit it when we introduce dynamic scenarios..